template/_package.json

```diff
@@ -140,10 +140,12 @@
     "jest": "^24.1.0",
     "vue-jest": "^3.0.3",
     <%_ } else if (test === 'ava') { _%>
-    "ava": "^1.0.1",
+    "@babel/preset-env": "^7.5.4",
```

Comment:
Any special reason for not using ava default config https://github.com/avajs/ava/blob/master/docs/recipes/babel.md#configuring-babel ?